node/test/unit/tessel.js

```diff
@@ -1261,14 +1277,14 @@ exports['Tessel.Pin'] = {
       this.a.pin[pinIndex].on('rise', spy);
       this.b.pin[pinIndex].on('rise', spy);
 
-      test.equal(this.a.pin[pinIndex].interruptMode, 'rise');
-      test.equal(this.b.pin[pinIndex].interruptMode, 'rise');
+      test.equal(this.a.pin[pinIndex].interruptMode, 'change');
+      test.equal(this.b.pin[pinIndex].interruptMode, 'change');
 
       // Simulate receipt of pin state changes
-      this.a.sock.read.returns(new Buffer([REPLY.ASYNC_PIN_CHANGE_N + pinIndex]));
+      this.a.sock.read.returns(new Buffer([(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (1 << 3)]));
       this.a.sock.emit('readable');
 
-      this.b.sock.read.returns(new Buffer([REPLY.ASYNC_PIN_CHANGE_N + pinIndex]));
+      this.b.sock.read.returns(new Buffer([(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (1 << 3)]));
       this.b.sock.emit('readable');
```

Comment:
> They will both fail because the mode is never actually set to rise or fall, it is set to change.

I think this means we need to step back for a moment and do a reality check by testing _all_ of the existing tessel modules to make sure this doesn't break them.